### DISCUSSION TITLE
[Projects Beta] No navigation bar in project view

### DISCUSSION THREAD

**Discussion by rickardp** [community/community#6463] [Projects and Issues]
Minor UX issue. It's annoying and quirky to have to use the browser back button to go back once entering a project. The old UX with the top bar (like the rest of GitHub) is more efficient. It's also quirky that you get a completely different nav experience in the new projects view.

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/6560

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
